inst/templates/ad_adrs.R

```diff
@@ -163,15 +196,17 @@ adrs <- adrs %>%
     )
   )
 
```

Comment:
@Sefan in old ADRS we printed the dataset with the result , but seems like we haven't done here right? for CBOR, BOR clinical benefit? is it possible to do that?

Reply:
I am not sure what you mean. In the templates the dataset is not printed.